cpp/src/Ice/SSL/SSLUtil.h

```diff
@@ -5,8 +5,8 @@
 #ifndef ICE_SSL_SSL_UTIL_H
 #define ICE_SSL_SSL_UTIL_H
 
+#include "DistinguishedName.h"
```

Comment:
Are you sure you need all the ICE_API in this file? It seems purely internal to the Ice core.

Reply:
I would check

Reply:
getSubjectName and encodeCertificate - Are used by Glacier2/IceGrid implementations
getSubjectAltNames - Don't need to be exported 
decodeCertificate and ScopedCertificate - are used by IceGrid implementation